RocketChat/Rocket.Chat.ReactNative
# [IMPROVE] Migrate from react-native-jitsi-meet to lib react-native-jitsimeet-custom

## Proposed changes

The migration will be necessary because of support of the lib react-native-jitsi-meet, with these new lib will be more easier to update our jitsi sdk, as the lib only initilize the jitsi sdk with parameters, with no changes on sdk.

The only problem is you need to upgrade your jitsi server, as they changed how the server work, and new sdk only work with new servers versions. (I'm not sure of the version, but it's pretty simple to update the Jitsi server)

## Issue(s)	

Closes: #4775

## How to test or reproduce

Enter in a call using iphone and try to see an user from web or mobile on your screen.
Use mobile server to test (https://mobile.rocket.chat)
## Screenshots

## Types of changes




## Checklist



## Further comments

Jira: [VC-57]

[VC-57]: https://rocketchat.atlassian.net/browse/VC-57?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
